cli/src/helper.rs

```diff
@@ -51,3 +55,41 @@ pub async fn bytes_from_hex_or_file(hex: Option<String>, file: Option<String>) -
         None
     })
 }
+
+pub async fn get_mnemonic() -> Result<String, Error> {
```

Comment:
I feel like `get` is a bit off 🤔 
It assumes it already exists, I think I would also call it `generate_mnemonic` tbh